indigo/analysis/toc/base.py

```diff
@@ -377,6 +388,21 @@ def title_without_type(self, item):
 
         return title
 
+    def get_def_element(self, element):
+        """ Uses the refersTo of an element containing a definition to find the matching <def> inside that element.
+            Returns the result of the xpath, which may be an empty list.
+        """
+        term = element.get('refersTo')
+        return element.xpath(f'.//a:def[@refersTo="{term}"]', namespaces={'a': self.act.namespace})
```

Comment:
To be pedantic, this returns a list, not a single element, so you may want to iterate over it here and return the first element. That will naturally return `None` to the caller if there is no element.